### PR TITLE
DM-28488: Document contents and usage of testdata_ci_hsc

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,39 +1,128 @@
...
+| i    | 10       | HSC-I           | 904010   |
++------+----------+-----------------+----------+
+| i    | 100      | HSC-I           | 904010   |
++------+----------+-----------------+----------+
+| i    | 1        | HSC-I           | 904014   |
++------+----------+-----------------+----------+
+| i    | 6        | HSC-I           | 904014   |
++------+----------+-----------------+----------+
+| i    | 12       | HSC-I           | 904014   |
++------+----------+-----------------+----------+
```
Comment (PaulPrice) [inline README.rst]: This feels like overkill, and a lot of duplication. If you must include something like this, a more compact format (e.g., list of detectors for each visit, with the corresponding filter) would be better.

File: README.rst [hunk 2]
```diff
@@ -1,39 +1,128 @@
...
+
+calibrations
+------------
+
+The ``calib/`` directory contains files for brighter-fatter, bias, dark, flat, sky frame, and defect correction, plus a database defining their validity ranges.
+
+The ``jointcal/`` directory contains jointcal output files from a full-focal plane run of jointcal astrometry and photometry.
+These files are used to apply an external jointcal calibrations during coadd processing, as there is not enough data in this package to reliably run jointcal on directly.
```
Comment (PaulPrice) [inline README.rst]: Singular `an` with plural `calibrations`.

File: README.rst [hunk 3]
```diff
@@ -1,39 +1,128 @@
...
+
+Reference catalog
+-----------------
+
+Reference catalogs for astrometry and photometry are included in the ``gaia_dr2_20200414/`` and ``ps1_pv3_3pi_20170110/`` directories, respectively.
+See the ``README.txt`` files in those directories for details.
+They are automatically loaded during the appropriate steps of ``ci_hsc``'s processing.
```
Comment (PaulPrice) [inline README.rst:67]: Point out that the reference catalogs contain only the shards of interest.

File: README.rst [hunk 4]
```diff
@@ -1,39 +1,128 @@
...
 Running the tests
 =================
 
 Set up the package
 ------------------
 
-This package provides the test data for the ``ci_hsc`` package.
-You must first setup this package, before setting up ``ci_hsc`` with the ``-k`` option, to keep this package set up::
+This package provides the test data for the `ci_hsc`_ package.
+You must first setup this package, before setting up `ci_hsc`_ with the ``-k`` option, to keep this data package set up::
```
Comment (PaulPrice) [inline README.rst]: This is confusing the actual requirements with one way of meeting those requirements. You don't need to do it exactly like you say. The actual requirement is that you must have both this package and `ci_hsc` configured in eups.

**Review by PaulPrice — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (parejkoj) [reply README.rst]: Ok, I've reworked it into a list per exposure. I've also added the butler command I used to get the info, for future reference.

**Pull request merged**